overlord/snapstate/handlers.go

```diff
@@ -1189,3 +1294,93 @@ func (m *SnapManager) doUnaliasV2(t *state.Task, _ *tomb.Tomb) error {
 	Set(st, snapName, snapst)
 	return nil
 }
+
+type otherDisabledAliases struct {
+	Disabled bool              `json:"disabled,omitempty"`
+	Manual   map[string]string `json:"manual,omitempty"`
+}
+
+func (m *SnapManager) doPreferAliasesV2(t *state.Task, _ *tomb.Tomb) error {
+	st := t.State()
+	st.Lock()
+	defer st.Unlock()
+	snapsup, snapst, err := snapSetupAndState(t)
+	if err != nil {
+		return err
+	}
+	snapName := snapsup.Name()
+
+	autoDisabled := snapst.AutoAliasesDisabled
+	if !autoDisabled {
```

Comment:
This may just look at snapst.AutoAliasesDisabled directly (probably side effect of refactorings).